src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final String DEFAULT_COLOR = "WHITE";
+    private static final int DEFAULT_VALUE = 10;
+    private static final int NUMBER_OF_FIGURES = 5;
+    private static final int MAX_VALUE = 15;
+    private final Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int number = random.nextInt(NUMBER_OF_FIGURES);
+        int first = random.nextInt(MAX_VALUE);
```

Comment:
bad naming
let's call it smth like `geometricProperty`
or even better create a method for it. Also, make sure this method can't return 0